packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -1,4 +1,31 @@
+:host([scale="s"]) {
+  --calcite-tile-content-gap: var(--calcite-spacing-sm);
```

Comment:
Should these be marked `--calcite-internal-x`? 

Anything intended to be public should have a doc / description. I'd think we want to "start small" with what we offer here, maybe just `--calcite-tile-heading-color` / `--calcite-tile-description-color`.

Reply:
> Should these be marked `--calcite-internal-x`?
> 
> Anything intended to be public should have a doc / description. I'd think we want to "start small" with what we offer here, maybe just `--calcite-tile-heading-color` / `--calcite-tile-description-color`.

Good question for @alisonailea and @SkyeSeitz.  If these are intended to eventually be "component" tokens specific to Tile, should these have the `internal` label included in their name?

Reply:
Decided to go the route of keeping this as simple as possible and only defining component tokens that are absolutely necessary for theming purposes only, mainly color.